tempreferences/expected.testCompareReferencesRenamedSequence.table

```diff
@@ -0,0 +1,5 @@
+MD5	Length	hg19mini.fasta	hg19mini_1renamed.fasta
```

Comment:
Remove the three files in the `tempreferences` directory from this PR using `git rm`